### PR TITLE
TrustedPlatformModule PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1869914](https://dev.azure.com/mseng/TechnicalContent/_queries/edit/1869914/)

### CODE REVIEW

File: docset/winserver2022-ps/trustedplatformmodule/Clear-Tpm.md [hunk 1]
```diff
@@ -82,6 +82,25 @@ SelfTest           : {191, 191, 245, 191...}
...
+ManagedAuthLevel   : Full
+OwnerAuth          :
+OwnerClearDisabled : True
+AutoProvisioning   : Disabled
+LockedOut          : False
+SelfTest           : {191, 191, 245, 191...}
+```
+
+This command resets the TPM by using the Physical Presence Interface (PPI).
+PPI does not use a value for owner authorization.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/trustedplatformmodule/Clear-Tpm.md:99]:
```suggestion
The PPI does not use a value for owner authorization.
```

File: docset/winserver2022-ps/trustedplatformmodule/Clear-Tpm.md [hunk 2]
```diff
@@ -82,6 +82,25 @@ SelfTest           : {191, 191, 245, 191...}
...
+
+Restart the system for the clear action to take effect.
+That restart might require user input to approve the clear request.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/trustedplatformmodule/Clear-Tpm.md:102]:
```suggestion
The restart might require user input to approve the clear request.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @TimShererWithAquent - kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
@tiburd I added the editorial changes. Ready for final review.

**Review by get-itips — CHANGES_REQUESTED** [review]
Hello @TimShererWithAquent Tim
Can you format the output like this (example) and also fix the other examples in this article?

image

**Review by dariomws — APPROVED** [review]
Agree with @get-itips.

**Comment by TimShererWithAquent** [comment]
@PatAltimore Please look at the feedback - I followed the existing examples for consistency. Is formatting on the previous examples in scope?

Also, do we have clarity on including the PS C:\> prompt? I put it in my example for consistency.

**Comment by TimShererWithAquent** [comment]
@get-itips - I implemented the formatting changes for the examples, but if you look at the preview, the second block does not appear as independent. Adding an extra blank line between the two blocks made no difference.

I tried to put a line of text between the two blocks and got a build error:

"Failed to load file: W:/sfqv-s/docset/winserver2022-ps/trustedplatformmodule/Clear-Tpm.md:35:(199) '```outputTpmReady           : FalseTpmPresent ...' Expect Heading, please try to use PlatyPS validate the content"

I'm guessing you can't have more than one fenced codeblock per example. 

Let me know how to proceed.

@dariomws @PatAltimore for awareness.

**Review by ghost — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
@tiburd We are at 3 approvals - please merge.

**Pull request merged**